pkg/authorization/api.go

```diff
@@ -31,6 +33,9 @@ type Request struct {
 	// RequestHeaders stores the raw request headers sent to the docker daemon
 	RequestHeaders map[string]string `json:"RequestHeaders,omitempty"`
 
+	// RequestPeerCertificates stores the request's TLS peer certificates
+	RequestPeerCertificates []*x509.Certificate `json:"RequestPeerCertificates,omitempty"`
```

Comment:
PEM should be fine as well, I'll rework this PR to encode those in PEM before sending them  over the wire